<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Quality</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>Close The Program After Complete</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLabel" name="FileType_2">
      <property name="text">
       <string>Loading...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="SaveLocationYV"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>File Info :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit2"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="FileType_3">
      <property name="text">
       <string>Loading...</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Shut Down After Complete</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="BrowseButtonYV">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="QualityBox"/>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="DownloadYoutubeVideo">
      <property name="styleSheet">
       <string notr="true">

QPushButton:hover {
  background-color: green;
  color: white;
}

QPushButton:pressed {
  transform: scale(0.95);
}
</string>
      </property>
      <property name="text">
       <string>DOWNLOAD</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">

QPushButton:hover {
  background-color: red;
  color: white;
}

QPushButton:pressed {
  transform: scale(0.95);
}
</string>
      </property>
      <property name="text">
       <string>CANCEL</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Duration</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
